tests/test_density_ComplexCF.py

```diff
@@ -83,10 +83,10 @@ def test_random_points(self):
         absolute_tolerance = 0.1
         # first bin is bad
         ocf.accumulate(freud.box.Box.square(box_size), points, comp,
-                       points, np.conj(comp))
+                       points, np.conj(comp), qargs={"exclude_ii": True})
```

Comment:
Given my other suggested change, perhaps it's also worth testing that the `(box, points, comp, values=np.conj(comp))` API also gives the same result.